CRFModuleValidation/Background_Survey.json

```diff
@@ -123,7 +123,7 @@
                                       "identifier": "days_per_week",
                                       "dataType": "integer",
                                       "prompt": "days per week",
-                                      "range": { "minimumValue": 0, "maximumValue": 7 },
+                                      "range": { "minimumValue": 1, "maximumValue": 7 },
```

Comment:
I think that we should leave the option of inputing `0`.  This is different from `nil`. Where `0` means "I didn't answer the first question correctly and now I realize that I should answer "no days", `nil` means "I am skipping the question". In this case, the skip button says "No vigorous physical activities", but this is really a less-than-ideal UX that matches what was already implemented (Android and iOS).  

If anything, would be better, I think, to use the number pad, by adding 

```
"uiHint" : "textfield"
```

(Default for a number field is apparently to preference the picker where the ui hint is undefined. The other way to change this would be to change ResearchSuite to default to a textfield for numbers.)

Reply:
...can you comment about this on https://sagebionetworks.jira.com/browse/IA-633? If we're trying to match the UX with Android they'll want to change it to a text field there too.